tools/dbtool.py

```diff
@@ -263,24 +276,25 @@ def fetch_files(express=False):
     if express:
         try:
             global express_enabled
-            sql_diffs = repo.commit(db_ver).diff(release_version,paths='sql/')
+            sql_diffs = repo.commit(db_ver).diff(release_version, paths=from_server_path('sql/'))
             if len(sql_diffs) > 0:
                 for diff in sql_diffs:
                     import_files.append(diff.a_path[4:])
```

Comment:
Need to add path here for express update imports. It reads them correctly but fails when it tries to actually import them.